scripts/stac/imagery/item.py

```diff
@@ -26,17 +28,59 @@
 class ImageryItem:
     stac: dict[str, Any]
 
-    def __init__(self, id_: str, now_string: str, stac_asset: STACAsset, stac_processing: STACProcessing) -> None:
+    def __init__(self, id_: str, stac_asset: STACAsset, stac_processing: STACProcessing) -> None:
         self.stac = {
             "type": "Feature",
             "stac_version": STAC_VERSION,
             "id": id_,
             "links": [Link(path=f"./{id_}.json", rel=Relation.SELF, media_type=StacMediaType.GEOJSON).stac],
             "assets": {"visual": {**stac_asset, "type": "image/tiff; application=geotiff; profile=cloud-optimized"}},
             "stac_extensions": [StacExtensions.file.value, StacExtensions.processing.value],
-            "properties": {"created": now_string, "updated": now_string, **stac_processing},
+            "properties": {"created": stac_asset["created"], "updated": stac_asset["updated"], **stac_processing},
         }
 
+    @classmethod
+    def from_file(cls, file_name: str) -> "ImageryItem":
+        """Create an ImageryItem from a file.
+
+        Args:
+            file_name: The s3 URL or local path of the file to load.
+
+        Returns:
+            ImageryItem: The new ImageryItem.
+        """
+        file_content = read(file_name)
+        stac_dict_from_s3 = json.loads(file_content.decode("UTF-8"))
+        if "bbox" in stac_dict_from_s3:  # tuple() does not serialise well into json
+            stac_dict_from_s3["bbox"] = tuple(stac_dict_from_s3["bbox"])
```

Comment:
Nit: We can use the walrus operator to avoid a lookup, like this:

```
        if (bbox := stac_dict_from_s3.get("bbox")) is not None:
            stac_dict_from_s3["bbox"] = tuple(bbox)
```

It *is* more complex syntax, though, so feel free to ignore.